scripts/set-default-contact/index.js

```diff
@@ -49,7 +49,7 @@ const getObjectFromMedicDb = async id => {
 const hasDefaultContact = async user => {
   if(typeof user.place === 'object' && typeof user.contact === 'object' &&  user.place._id  ){
     const place = await getObjectFromMedicDb(user.place._id);
-    return !(typeof place.contact === 'object' && !place.contact._id);
+    return !((typeof place.contact === 'object' && !place.contact._id) || typeof place.contact !== 'object');
```

Comment:
TLDR: I think we can simplify this logic:
```js
    return typeof place.contact === 'object' && place.contact._id;
```

Steps:
- We start with:
  - `!((typeof place.contact === 'object' && !place.contact._id) || typeof place.contact !== 'object')`
- Use DeMorgan's law to distribute the outer `!`:
  - `!(typeof place.contact === 'object' && !place.contact._id) && typeof place.contact === 'object'`
- Use DeMorgan's law again on the inner portion:
  - `(typeof place.contact !== 'object' || place.contact._id) && typeof place.contact === 'object'`
- Reduce since we cannot have both !x AND x:
  - `place.contact._id && typeof place.contact === 'object'`

Reply:
Love this!  That line made me feel a little nauseous - this approach is wonderfully clarifying. Thanks for taking the time to walk me through it.

Follow up below